src/solvers/imex_ark.jl

```diff
@@ -53,60 +55,84 @@ function step_u!(integrator, cache::IMEXARKCache)
     s = length(b_exp)
 
     if !isnothing(T_imp!) && !isnothing(newtons_method)
-        update!(
-            newtons_method,
-            newtons_method_cache,
-            NewTimeStep(t),
-            jacobian -> isnothing(γ) ? sdirk_error(name) : T_imp!.Wfact(jacobian, u, p, dt * γ, t),
-        )
+        @nvtx "update!" color = colorant"yellow" begin
+            update!(
+                newtons_method,
+                newtons_method_cache,
+                NewTimeStep(t),
+                jacobian -> isnothing(γ) ? sdirk_error(name) : T_imp!.Wfact(jacobian, u, p, dt * γ, t),
+            )
+        end
     end
 
     for i in 1:s
```

Comment:
Maybe add one for each stage, e.g.
```suggestion
    for i in 1:s
        NVTX.@range "stage" payload=i begin
```